src/api/dexPriceEstimator/DexPriceEstimatorApi.ts

```diff
@@ -22,6 +23,25 @@ interface OrderBookParams {
   hops?: number
 }
 
+/**
+ * Price point as defined in the API
+ * Both price and volume are numbers (floats)
+ *
+ * The price and volume are expressed in atoms
+ */
+export interface RawPricePoint {
```

Comment:
Maybe we should rename to `Quote` or `PriceQuote`?

Reply:
it could be confused with price for Quote Token then

Reply:
Okok, can we at least drop the Raw?
raw for me made sense in the chart context (it was private in that module). But if you make it a general exported data type, all data is raw